.github/workflows/tag.yml

```diff
@@ -46,8 +48,37 @@ jobs:
         echo "new_tag=${script_output}"  >> "$GITHUB_OUTPUT"
       shell: bash {0}
 
-  auto-semver-tag:
-    needs: auto-git-tag
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: eu-west-1
+
+    - name: Login to Amazon ECR
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1.6.1
```

Comment:
Theres a v2 release with some security updates that might be worth trying out

Reply:
Great to know - let me whack that in now!